templates/bot/ts/notification-restify/src/index.ts

```diff
@@ -9,17 +9,58 @@ import { server } from "./internal/server";
 server.post("/api/notification", async (req, res) => {
   for (const target of await ConversationBot.installations()) {
     await target.sendAdaptiveCard(
-      buildAdaptiveCard(() => {
-        return {
+      buildAdaptiveCard(
+        {
           title: "New Event Occurred!",
           appName: "Contoso App Notification",
           description: `This is a sample http-triggered notification to ${target.type}`,
           notificationUrl: "https://www.adaptivecards.io/",
-        };
-      }, notificationTemplate)
+        },
+        notificationTemplate
+      )
     );
   }
 
+  /****** To distinguish different target types ******/
+  /** "Channel" means this bot is installed to a Team (default to notify General channel)
+  if (target.type === "Channel") {
+    // Directly notify the Team (to the default General channel)
+    await target.sendAdaptiveCard(...);
+
+    // List all channels in the Team then notify each channel
+    const channels = await target.channels();
+    for (const channel of channels) {
+      await channel.sendAdaptiveCard(...);
+    }
+
+    // List all members in the Team then notify individual person
```

Comment:
```suggestion
    // List all members in the Team then notify each member
```
To be consistent with above comments